exporter/metric/README.md

```diff
@@ -42,16 +51,18 @@ defer pusher.Stop()
 ctx := context.Background()
 meter := pusher.Provider().Meter("cloudmonitoring/example")
 
-counter := metric.Must(meter).NewInt64Counter("counter-foo")
-labels := []kv.KeyValue{kv.Key("key").String("value")}
+counter := metric.Must(meter).NewInt64Counter("counter.foo")
```

Comment:
Any specific reason for replacing `-` with `.` in the metric name?

Google Cloud Monitoring is actually using `/` and `_` as separators normally, e.g. see metrics in https://cloud.google.com/monitoring/api/metrics_kubernetes

Reply:
I'd thought I'd seen `counter.foo` used in another example and so inadvertently copied. Will revert.

Reply:
Reverted: 0fee3ade2a5087fdfaf7df92d8577c358e499175